doc/embedded/supported_boards.md

```diff
@@ -0,0 +1,171 @@
+| C906      | [Bootlin toolchain link](https://toolchains.bootlin.com/releases_riscv64-lp64d.html)| [Sysroot and toolchain prefix](#C906) | [C906 on riscv](https://riscv.org/news/2020/11/xuantie-c906-based-allwinner-risc-v-processor-to-power-12-linux-sbcs/attachment/allwinner-xuantie-c906-risc-v-processor/) |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| x280      | [Bootlin toolchain link](https://toolchains.bootlin.com/releases_riscv64-lp64d.html)| [Sysroot and toolchain prefix](#x280) | [x280 on SiFive](https://www.sifive.cn/api/document-file?uid=x280-datasheet) |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| KATAMI    | [Bootlin toolchain link](https://toolchains.bootlin.com/releases_x86-i686.html)     | [Sysroot and toolchain prefix](#KATAMI) |[Pentium 3 on Wikipedia](https://en.wikipedia.org/wiki/Pentium_III)          |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NORTHWOOD | [Bootlin toolchain link](https://toolchains.bootlin.com/releases_x86-64.html)       | [Sysroot and toolchain prefix](#NORTHWOOD)   |[Pentium 4 on Wikipedia](https://en.wikipedia.org/wiki/Pentium_4)       |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| COPPERMINE | [Bootlin toolchain link](https://toolchains.bootlin.com/releases_x86-i686.html)   | [Sysroot and toolchain prefix](#COPPERMINE) |[Pentium 3 on Wikipedia](https://en.wikipedia.org/wiki/Pentium_III)       |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+
+If you didn't see your architecture in the table above, use the closest
+architecture with a similar word size, or, adapt the parameters directly in
+`CMake/crosscompile-arch-config.cmake` and feel free to open a pull request so we can get
+the new architecture added to this table.
+
+### ARM11
+
+```
+-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv6-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf-
+-DCMAKE_SYSROOT=/path/to/bootlin/toolchain/armv6-eabihf--glibc--stable-2024.02-1/arm-buildroot-linux-gnueabihf/sysroot
+```
+
+### CORTEXA7 
+
+```-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf-
```

Comment:
```suggestion
```
-DTOOLCHAIN_PREFIX=/path/to/bootlin/toolchain/armv7-eabihf--glibc--stable-2024.02-1/bin/arm-buildroot-linux-gnueabihf-
```
(Just a little consistency thing, it is super minor.  It renders the same either way)